source/_components/pcal9535a.markdown

```diff
@@ -0,0 +1,130 @@
+---
+title: "PCAL9535A I2C GPIO expander"
+description: "Instructions on how to integrate the PCAL9535A GPIO pin expander with I2C interface into Home Assistant."
+logo: raspberry-pi.png
+ha_category:
+  - DIY
+  - Binary Sensor
+  - Switch
+ha_release: 0.101
+ha_iot_class: Local Polling
+---
+
+The `pcal9535a` integration is the base for all related pcal9535a platforms in Home Assistant. There is no setup needed for the integration itself, for the platforms please check their corresponding sections.
+
+One of the use cases is [Seeed studio Raspberry Pi Relay Board](http://wiki.seeedstudio.com/Raspberry_Pi_Relay_Board_v1.0/).
+
+For more details about the PCAL9535A I2C I/O port expander you can find its datasheet here: [PCAL9535A](https://www.nxp.com/docs/en/data-sheet/PCAL9535A.pdf).
+
+## Binary Sensor
+
+The `pcal9535a` binary sensor platform allows you to read sensor values from the I/O pins of your [PCAL9535A I2C I/O expander](https://www.nxp.com/products/interfaces/ic-spi-serial-interface-devices/ic-general-purpose-i-o/low-voltage-16-bit-ic-bus-i-o-port-with-interrupt-and-agile-i-o:PCAL9535A).
+
+The pin numbers are from 0 to 15 where: 0-7 correspond to port P0 (P0_0 - P0_7) and 8-15 to port P1 (P1_0 - P1_7).
```

Comment:
```suggestion
The pin numbers are from 0 to 15, where: 0-7 corresponds to port P0 (P0_0 - P0_7) and 8-15 to port P1 (P1_0 - P1_7).
```

Reply:
Are you sure it should be singular? We are talking about several pins (0 to 7) which correspond to the port.

Reply:
Sorry, you are right, adjusted suggestion.

Reply:
I'm changing it for now as requested, but I still believe it might be wrong:) However English is not my native language.

Reply:
Fixed